cmasher/utils.py

```diff
@@ -150,7 +154,7 @@ def _get_cmap_lightness_rank(cmap: CMAP) -> tuple[int, int, float, float, float,
             L_rmse = np.max(
                 [
                     np.around(np.std(derivs[: central_i[0]]), 0),
-                    np.around(np.std(derivs[central_i[1] :]), 0),
+                    np.around(np.std(derivs[central_i[1]:]), 0),
```

Comment:
please refrain from applying purely stylistic changes, as it messes up with `git blame` and makes reviewing harder.

Reply:
Sure thing! I would revert unrelated stylistic changes from this PR.